docs/sphinx/source/index.rst

```diff
@@ -0,0 +1,19 @@
+Welcome to TUF's documentation!
+===============================
+
+This is the documentation of the **reference implementation** of
+`The Update Framework (TUF) <https://theupdateframework.github.io/>`_.
+
+The reference implementation strives to be a readable guide and demonstration
+for those working on implementing TUF in their own languages, environments, or
+update systems. It is written in Python and intended to conform to version 1.0 of the
+`TUF specification <https://theupdateframework.github.io/specification/latest/>`_.
+
+
+.. toctree::
+   :maxdepth: 2
+   :caption: Contents:
+   
+   modern-implementation
+   legacy-implementation
```

Comment:
I suggest we call "modern-implementation" just "API reference" (with the expectation that we will add "Tutorials", "Examples" etc to this list soon) and then we have an additional explanation about the legacy code: I think that explanation can be right here on this page, I see no reason to have multiple single paragraph pages.

Reply:
I suggest we do not include the legacy implementation. We do not plan to support it.

Reply:
yes exactly, I skipped over that bit in my brain :)